README.md

```diff
@@ -138,6 +139,7 @@ The following table shows the supported combinations:
 
 | LiT               | LND          | Loop         | Faraday       | Pool         | Taproot Assets |
 |-------------------|--------------|--------------|---------------|--------------|----------------|
+| **v0.10.1-alpha** | v0.16.2-beta | v0.24.1-beta | v0.2.11-alpha | v0.6.3-beta  | v0.2.0-alpha   |
```

Comment:
I think we might want to wait for `lnd v0.16.3-beta` as that would otherwise require yet another release? Or is this urgent enough on its own?